css/interface.css

```diff
@@ -992,6 +992,10 @@ header .btn,
   width: 18%;
 }
 
+.detail-table-panels-holder .image-type-select .rTableCell.title {
+  width: 20.6%;
```

Comment:
@YaroslavOvdii Why 20.6% specifically?

Reply:
@YaroslavOvdii That doesn't explain why it's `20.6%`. How was this calculated?

Reply:
@tonytlwu remade this solution without `20.6%`